scripts/ann-benchmarks/get_dataset.py

```diff
@@ -0,0 +1,90 @@
+#
+# Copyright (c) 2023, NVIDIA CORPORATION.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+import argparse
+import os
+import subprocess
+from urllib.request import urlretrieve
+
+
+def get_dataset_path(name, ann_bench_data_path):
+    if not os.path.exists(ann_bench_data_path):
+        os.mkdir(ann_bench_data_path)
+    return os.path.join(ann_bench_data_path, f"{name}.hdf5")
+
+
+def download_dataset(url, path):
+    if not os.path.exists(path):
+        print(f"downloading {url} -> {path}...")
+        urlretrieve(url, path)
+
+
+def convert_hdf5_to_fbin(path, normalize):
+    ann_bench_scripts_path = os.path.join(os.getcwd(), "cpp/bench/ann/scripts",
```

Comment:
Might be worthwhile to make the hard-coded path into a constant. Or better yet, into a variable that can be overridden with a command-line argument.